### DISCUSSION TITLE
VPN w Azure

### DISCUSSION THREAD

**Discussion by szympulka** [AzureCommunityPL/group#475] [Pomoc]
Chciałbym połączyć się z komputerem z biura za pomocą VPN i logowania po AAD/login&pass. Czy Azure ma usługę która na to pozwala? Czy VPN Gateway jest tym czego szukam? A może muszę mieć wirtualną maszynę w biurze? 

Skala to małe biuro dlatego szukam jakiegoś gotowego i łatwego rozwiązania

**Comment by JaroslawBakInitius** [comment]
Generalnie odpowiedź brzmi tak
Jeśli to komputer w biurze, to musisz mieć spięte biuro z Azurem S2S VPN
Jeśli ma to być logowanie przez AAD to minimum usługa w Planie 1

--
z poważaniem
Jarosław Bąk
kom.
+48 501 507 264


From: Szymon Pulka ***@***.***>
Sent: Friday, January 13, 2023 12:48 PM
To: AzureCommunityPL/group ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [AzureCommunityPL/group] VPN w Azure (Discussion #475)


Chciałbym połączyć się z komputerem z biura za pomocą VPN i logowania po AAD/login&pass. Czy Azure ma usługę która na to pozwala? Czy VPN Gateway jest tym czego szukam? A może muszę mieć wirtualną maszynę w biurze?

Skala to małe biuro dlatego szukam jakiegoś gotowego i łatwego rozwiązania

**Reply by szympulka** [reply]
Czyli dobrze poszukałem. Czy wtedy na routerze mogę też zastosować VPN przez login i hasło?

**Reply by eklime** [reply]
> Czyli dobrze poszukałem. Czy wtedy na routerze mogę też zastosować VPN przez login i hasło?

Na routerze zapinasz S2S IPSec, więc potrzebujesz IP VPN Gateway i PreSharedKey który podajesz na obu urzedzeniach brzegowych.
